index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="de">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <title>Typing-Practice</title>
+  </head>
+  <body>
+    <div class="body__welcome-container">
```

Comment:
```suggestion
    <div class="welcome-container">
```

Do not use the HTML-Tag as part of the name ;)